server/mediaserver.go

```diff
@@ -234,7 +235,7 @@ func gotRTMPStreamHandler(s *LivepeerServer) func(url *url.URL, rtmpStrm stream.
 			}
 
 			getReusableBroadcast := func(b *common.DBJob) (*ethTypes.Job, bool) {
```

Comment:
We actually should remove this entire function `getReusableBroadcast` and any surrounding references to it.